### PR TITLE
JW-1211 release agent

### PR DESCRIPTION
**Pull request by michal-clapinski**

### CODE REVIEW

File: include/windows_types.h [hunk 1]
```diff
@@ -27,6 +27,8 @@ typedef BOOLEAN bool;
 
 #define __LITTLE_ENDIAN_BITFIELD
 
+#define ETH_HLEN 14
```
Comment (PawelMarchewkaCodiLime) [inline include/windows_types.h]: What is the reason of this change ? Is this already provided by one of the following windows headers ?

Comment (PawelMarchewkaCodiLime) [inline include/windows_types.h]: What is the reason of this change ? Is this already provided by one of the following windows headers ?

Comment (michal-clapinski) [reply include/windows_types.h]: It's already provided in windowsstubs and it was generating a warning.
